### PR TITLE
Feat/#3 setting up db tables

### PR DESCRIPTION
**Pull request by sganous**
Closes issue #3.

### CODE REVIEW

File: db/migrate/20180103042254_create_starting_tables.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      t.string "city", null: false
+      t.string "district"
+      t.string "postal_code"
+      t.string "country", null: false
+      t.integer "company_id", null: false
+      t.timestamps
+    end
+    end
+  end
+end
```
Comment (S1v4) [inline db/migrate/20180103042254_create_starting_tables.rb]: Add a line between each `create_table` and there's also an extra `end` on line 43 that should be on line 33.  We also need resume and cover letter on the applicant table [guide](https://rubyplus.com/articles/3821-File-Uploads-using-Paperclip-in-Rails-5).

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  create_table "companies", force: :cascade do |t|
+    t.string "name", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "logo"
+    t.string "#<ActiveRecord::ConnectionAdapters::PostgreSQL::TableDefinition"
```
Comment (S1v4) [inline db/schema.rb]: I'm not sure what this field is `#<ActiveRecord::ConnectionAdapters::PostgreSQL::TableDefinition`. I haven't seen it before and it's only on our two initial tables. Also, logo needs to be an `t.attachment :photo`.

File: db/schema.rb [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  create_table "users", force: :cascade do |t|
+    t.string "first_name", null: false
+    t.string "last_name", null: false
+    t.string "username", null: false
+    t.boolean "is_admin", default: false, null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.string "email"
+    t.string "#<ActiveRecord::ConnectionAdapters::PostgreSQL::TableDefinition"
```
Comment (S1v4) [inline db/schema.rb]: same here with this field.

**Comment by sganous** [comment]
Issues addressed and fixed.  Was initially thinking about storing documents like images, resumes, and cover letters in a file structure but 'paperclip' provides a nice functionality.  We can discuss the positives and negatives of file structure vs db but for now lets see how this goes and adjust if we run into any problems.

**Pull request merged**